src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import duke.task.Deadline;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskListTest {
+    private final TaskList taskList = new TaskList();
+    @Test
+    public void addTest() {
```

Comment:
```suggestion
    public void add_deadline_successfullyAddedToTaskList() {
```
Can consider renaming test methods to be more descriptive 😄

Reply:
Done!